DPL: move away from CallbackRegistry

ktf opened the pull request:
The recursive template is simply slow to compile, accounting alone for 10% of the
compilation time in analysis. While this version does not allow
to invoke callbacks from the id anymore, we never used such a feature,
so we can simply drop it.

alibuild commented:
Error while checking build/O2/fullCI for a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf at 2022-08-31 12:48:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/9731-slc8_x86-64/0/Framework/GUISupport/test/test_CustomGUIGL.cxx:58:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
/sw/SOURCES/O2/9731-slc8_x86-64/0/Framework/GUISupport/test/test_CustomGUISokol.cxx:60:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
/sw/SOURCES/O2/9731-slc8_x86-64/0/Framework/GUISupport/test/test_SimpleTracksED.cxx:51:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9731/a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf/build_O2_fullCI/pretty.html).

alibuild commented:
Error while checking build/O2/o2 for a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf at 2022-08-31 13:20:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/9731-slc7_x86-64/0/Framework/GUISupport/test/test_CustomGUIGL.cxx:58:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
/sw/SOURCES/O2/9731-slc7_x86-64/0/Framework/GUISupport/test/test_CustomGUISokol.cxx:60:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
/sw/SOURCES/O2/9731-slc7_x86-64/0/Framework/GUISupport/test/test_SimpleTracksED.cxx:51:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9731/a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf/build_O2_o2/pretty.html).

alibuild commented:
Error while checking build/O2/o2-dataflow-cs8 for a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf at 2022-09-19 04:27:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/9731-slc8_x86-64/0/Framework/GUISupport/test/test_CustomGUIGL.cxx:58:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
/sw/SOURCES/O2/9731-slc8_x86-64/0/Framework/GUISupport/test/test_CustomGUISokol.cxx:60:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
/sw/SOURCES/O2/9731-slc8_x86-64/0/Framework/GUISupport/test/test_SimpleTracksED.cxx:51:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9731/a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf/build_O2_o2-dataflow-cs8/pretty.html).

alibuild commented:
Error while checking build/AliceO2/O2/o2/macOS-arm for a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf at 2022-08-31 13:56:
```
## sw/BUILD/O2-latest/log
 15/443 Test  #16: Framework/Utils/test/test_DPLOutputTest.cxx ..........................................***Failed    0.86 sec
[ERROR] pid 65544 (Generator) crashed with 1
[ERROR] pid 65545 (Sink) crashed with 128
[ERROR] pid 65543 (internal-dpl-clock) crashed with 128
[ERROR] SEVERE: Device internal-dpl-clock (65543) returned with 128
 16/443 Test  #15: Framework/Utils/test/test_DPLBroadcasterMerger.cxx ...................................***Failed    0.92 sec
[ERROR] pid 65534 (Broadcaster) crashed with 1
[ERROR] pid 65536 (pip1) crashed with 128
[ERROR] pid 65535 (pip0) crashed with 128
[ERROR] pid 65532 (internal-dpl-clock) crashed with 128
[ERROR] pid 65538 (Merger) crashed with 128
[ERROR] pid 65533 (Generator) crashed with 128
[ERROR] pid 65539 (Sink) crashed with 128
[ERROR] SEVERE: Device internal-dpl-clock (65532) returned with 128
 19/443 Test  #18: Framework/Utils/test/test_RootTreeReader.cxx .........................................***Failed    0.84 sec
[ERROR] pid 65617 (sink) crashed with 1
[ERROR] pid 65615 (internal-dpl-clock) crashed with 128
[ERROR] pid 65616 (source) crashed with 128
[ERROR] SEVERE: Device internal-dpl-clock (65615) returned with 128
 21/443 Test  #17: Framework/Utils/test/test_RootTreeWriterWorkflow.cxx .................................Subprocess aborted***Exception:   1.39 sec
[ERROR] pid 65600 (internal-dpl-clock) crashed with 1
[ERROR] pid 65605 (sink2) crashed with 128
[ERROR] pid 65604 (sink1) crashed with 128
[ERROR] pid 65603 (source) crashed with 1
[ERROR] SEVERE: Device internal-dpl-clock (65600) returned with 1
Error in <TFile::TFile>: file /tmp/test_RootTreeWriter.root does not exist
Error in <TFile::TFile>: file /tmp/test_RootTreeWriter_alt.root does not exist
99% tests passed, 4 tests failed out of 443
 77/102 Test  #91: test_Framework_test_CallbackService .....................***Failed    0.33 sec
[ERROR] pid 97345 (dest) crashed with 1
[ERROR] pid 97344 (source) crashed with 1
[ERROR] pid 97343 (internal-dpl-clock) crashed with 1
[ERROR] SEVERE: Device internal-dpl-clock (97343) returned with 1
 78/102 Test  #92: test_Framework_test_RegionInfoCallbackService ...........***Failed    0.33 sec
[ERROR] pid 97349 (dest) crashed with 1
[ERROR] pid 97348 (source) crashed with 1
[ERROR] pid 97347 (internal-dpl-clock) crashed with 1
[ERROR] SEVERE: Device internal-dpl-clock (97347) returned with 1
 80/102 Test  #94: test_Framework_test_DanglingOutputs .....................***Failed    0.33 sec
[ERROR] pid 97354 (internal-dpl-injected-dummy-sink) crashed with 1
[ERROR] pid 97353 (B) crashed with 1
[ERROR] pid 97352 (A) crashed with 1
[ERROR] pid 97351 (internal-dpl-clock) crashed with 1
[ERROR] SEVERE: Device internal-dpl-clock (97351) returned with 1
 81/102 Test  #95: test_Framework_test_DataAllocator .......................***Failed    0.33 sec
[ERROR] pid 97395 (spectator-sink) crashed with 1
[ERROR] pid 97394 (sink) crashed with 1
[ERROR] pid 97393 (source) crashed with 1
[ERROR] pid 97392 (internal-dpl-clock) crashed with 1
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9731/a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf/build_AliceO2_O2_o2_macOS-arm/pretty.html).

alibuild commented:
Error while checking build/AliceO2/O2/o2/macOS for a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf at 2022-09-17 21:34:
```
## sw/BUILD/O2-latest/log
  5/449 Test  #15: Framework/Utils/test/test_DPLBroadcasterMerger.cxx ...................................***Failed    0.39 sec
[ERROR] pid 14302 (internal-dpl-clock) crashed with 1
[ERROR] pid 14308 (Sink) crashed with 128
[ERROR] pid 14307 (Merger) crashed with 128
[ERROR] pid 14306 (pip1) crashed with 128
[ERROR] pid 14305 (pip0) crashed with 128
[ERROR] pid 14304 (Broadcaster) crashed with 128
[ERROR] pid 14303 (Generator) crashed with 128
[ERROR] SEVERE: Device internal-dpl-clock (14302) returned with 1
  9/449 Test  #16: Framework/Utils/test/test_DPLOutputTest.cxx ..........................................***Failed    0.35 sec
[ERROR] pid 14313 (internal-dpl-clock) crashed with 1
[ERROR] pid 14315 (Sink) crashed with 128
[ERROR] pid 14314 (Generator) crashed with 128
[ERROR] SEVERE: Device internal-dpl-clock (14313) returned with 1
 10/449 Test  #18: Framework/Utils/test/test_RootTreeReader.cxx .........................................***Failed    0.33 sec
[ERROR] pid 14320 (internal-dpl-clock) crashed with 1
[ERROR] pid 14322 (sink) crashed with 128
[ERROR] pid 14321 (source) crashed with 128
[ERROR] SEVERE: Device internal-dpl-clock (14320) returned with 1
 16/449 Test  #17: Framework/Utils/test/test_RootTreeWriterWorkflow.cxx .................................Subprocess aborted***Exception:   1.14 sec
[ERROR] pid 14316 (internal-dpl-clock) crashed with 1
[ERROR] pid 14318 (sink1) crashed with 128
[ERROR] pid 14317 (source) crashed with 128
[ERROR] pid 14319 (sink2) crashed with 128
[ERROR] SEVERE: Device internal-dpl-clock (14316) returned with 1
Error in <TFile::TFile>: file /tmp/test_RootTreeWriter.root does not exist
Error in <TFile::TFile>: file /tmp/test_RootTreeWriter_alt.root does not exist
 80/449 Test  #90: CCDB/test/testCcdbApi_ConfigParam.cxx ................................................***Failed    1.92 sec
[ERROR] Requested resource does not exist: http://ccdb-test.cern.ch:8080/GLO/Config/PVertexer/1663442715794/
/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9731/0/CCDB/test/testCcdbApi_ConfigParam.cxx:131: [1;31;49merror: in "testConfigParamRetrieval": check object != nullptr has failed[0;39;49m
/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9731/0/CCDB/test/testCcdbApi_ConfigParam.cxx:133: [1;31;49merror: in "testConfigParamRetrieval": check object->getMemberProvenance("useMeanVertexConstraint") == o2::conf::ConfigurableParam::EParamProvenance::kCCDB has failed[0;39;49m
/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9731/0/CCDB/test/testCcdbApi_ConfigParam.cxx:135: [1;31;49merror: in "testConfigParamRetrieval": check p1.getMemberProvenance("useMeanVertexConstraint") == o2::conf::ConfigurableParam::EParamProvenance::kCCDB has failed[0;39;49m
/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9731/0/CCDB/test/testCcdbApi_ConfigParam.cxx:136: [1;31;49merror: in "testConfigParamRetrieval": check object == &p1 has failed[0;39;49m
99% tests passed, 5 tests failed out of 449
 75/101 Test  #89: test_Framework_test_CallbackService .....................***Failed    0.16 sec
[ERROR] pid 80971 (internal-dpl-clock) crashed with 1
[ERROR] pid 80973 (dest) crashed with 1
[ERROR] pid 80972 (source) crashed with 1
[ERROR] SEVERE: Device internal-dpl-clock (80971) returned with 1
 76/101 Test  #90: test_Framework_test_RegionInfoCallbackService ...........***Failed    0.16 sec
[ERROR] pid 80975 (internal-dpl-clock) crashed with 1
[ERROR] pid 80977 (dest) crashed with 1
[ERROR] pid 80976 (source) crashed with 1
[ERROR] SEVERE: Device internal-dpl-clock (80975) returned with 1
 78/101 Test  #92: test_Framework_test_DanglingOutputs .....................***Failed    0.16 sec
[ERROR] pid 80980 (A) crashed with 1
[ERROR] pid 80982 (internal-dpl-injected-dummy-sink) crashed with 1
[ERROR] pid 80981 (B) crashed with 1
[ERROR] pid 80979 (internal-dpl-clock) crashed with 1
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9731/a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf/build_AliceO2_O2_o2_macOS/pretty.html).

alibuild commented:
Error while checking build/O2/o2-dataflow for a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf at 2022-09-07 06:17:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/9731-slc7_x86-64/0/Framework/GUISupport/test/test_CustomGUISokol.cxx:60:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
/sw/SOURCES/O2/9731-slc7_x86-64/0/Framework/GUISupport/test/test_CustomGUIGL.cxx:58:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
/sw/SOURCES/O2/9731-slc7_x86-64/0/Framework/GUISupport/test/test_SimpleTracksED.cxx:51:45: error: 'ClockTick' is not a member of 'o2::framework::CallbackService::Id'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9731/a6a9ed662dba224b8d41e022e81f93bb8e1fd9bf/build_O2_o2-dataflow/pretty.html).

ktf committed:
DPL: drop CallbackRegistry

Replace the callback registry with some compatible code which
does not need the compile-time recursion and which complains if
we invoke a callback with the wrong API. This
also simplifies possibility to have more than one callback being set
in the future.

ktf commented:
@costing @Barthelemy the multinode issue is back. Did anything change?

```
## sw/BUILD/QualityControl-latest/log
27/39 Test #33: multinode_test ..........................***Failed   35.75 sec
97% tests passed, 1 tests failed out of 39
```

Barthelemy commented:
@knopers8 see previous comment from Giulio. @ktf is it something you saw in many places or just here ?

knopers8 commented:
> Did anything change?

Not in QC.

alibuild commented:
Error while checking build/O2/fullCI for f1f4ac7df14b0cf262f53546e2ff89c4e0bc2004 at 2022-10-28 03:18:
```
## sw/BUILD/o2checkcode-latest/log
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/Common/test/testGPUsortCUDA.cu:22:10: error: 'boost/test/unit_test.hpp' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx:37:10: error: 'omp.h' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUTracking/Base/GPUReconstruction.cxx:37:10: error: 'omp.h' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUTracking/Base/GPUReconstructionCPU.cxx:45:10: error: 'omp.h' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUTracking/display/GPUDisplay.cxx:36:10: error: 'omp.h' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUTracking/Base/cuda/GPUReconstructionCUDAGenRTC.cu:16:10: error: 'omp.h' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/cuda/../Shared/Utils.h:26:10: error: 'boost/program_options.hpp' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/cuda/../Shared/Utils.h:26:10: error: 'boost/program_options.hpp' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/../Shared/Utils.h:146:34: error: use of undeclared identifier 'int4'; did you mean 'int'? [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:64:8: error: unknown type name '__host__' [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:64:27: error: expected ';' after top level declarator [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:64:33: error: overloaded 'operator+=' must have at least one parameter of class or enumeration type [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:64:44: error: unknown type name 'int4'; did you mean 'int'? [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:64:53: error: unknown type name 'int4'; did you mean 'int'? [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:66:4: error: member reference base type 'int' is not a structure or union [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:66:11: error: member reference base type 'int' is not a structure or union [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:67:4: error: member reference base type 'int' is not a structure or union [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:67:11: error: member reference base type 'int' is not a structure or union [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:68:4: error: member reference base type 'int' is not a structure or union [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:68:11: error: member reference base type 'int' is not a structure or union [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:69:4: error: member reference base type 'int' is not a structure or union [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:69:11: error: member reference base type 'int' is not a structure or union [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:85:1: error: unknown type name '__global__' [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:90:19: error: use of undeclared identifier 'blockIdx' [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:90:32: error: use of undeclared identifier 'blockDim' [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:90:45: error: use of undeclared identifier 'threadIdx' [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/Kernels.hip.cxx:90:78: error: use of undeclared identifier 'blockDim' [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/Framework/Logger/include/Framework/Logger.h:14:10: error: 'fairlogger/Logger.h' file not found [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:520:12: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:1059:69: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:1175:5: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:1852:16: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:2276:18: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:4336:16: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/sm_20_atomic_functions.h:89:39: error: redefinition of 'atomicAdd' [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/Framework/Logger/include/Framework/Logger.h:14:10: error: 'fairlogger/Logger.h' file not found [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:520:12: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:1059:69: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:1175:5: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:1852:16: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:2276:18: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/cuda/std/detail/libcxx/include/type_traits:4336:16: error: CUDA device code does not support variadic functions [clang-diagnostic-error]
/usr/local/cuda-11.7/include/sm_20_atomic_functions.h:89:39: error: redefinition of 'atomicAdd' [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/../Shared/Utils.h:146:34: error: use of undeclared identifier 'int4'; did you mean 'int'? [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/GPU/GPUbenchmark/hip/benchmark.hip.cxx:199:35: error: use of undeclared identifier 'int4'; did you mean 'int'? [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALCalibExtractor.h:36:10: error: 'omp.h' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALCalibExtractor.h:36:10: error: 'omp.h' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALCalibExtractor.h:36:10: error: 'omp.h' file not found [clang-diagnostic-error]
/sw/SOURCES/O2/9731-slc8_x86-64/0/Detectors/TOF/calibration/src/TOFChannelCalibrator.cxx:23:10: error: 'omp.h' file not found [clang-diagnostic-error]
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9731/f1f4ac7df14b0cf262f53546e2ff89c4e0bc2004/build_O2_fullCI/pretty.html).
